config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: quay.io/alanconway/origin-cluster-logging-operator
```

Comment:
This is one of the changes we need to watch out and never commit.

Reply:
This is tricky, I'll think about it. I really hate how kustomize and operator-sdk fail to distinguish between source and generated files. We could reset images names in a "pre-commit" target but people can forget to use it. We could have a not-checked-in set of "shadow" directories for dev work, but then kustomize changes that should be checked in may get lost. Hmm.